src/lib/Scenes/Artwork/Components/AuctionCountDownTimer.tsx

```diff
@@ -60,7 +24,7 @@ export class AuctionCountDownTimer extends React.Component<AuctionCountDownTimer
         labelWithoutTimeRemaining={sale.formattedStartDateTime}
         labelFontSize="2"
         timerFontSize="4t"
-        countdownEnd={auctionStatus === "hasNotStarted" ? sale.startAt : sale.endAt}
+        countdownEnd={auctionState === "isPreview" ? sale.startAt : sale.endAt}
```

Comment:
Looking at this now... how does this deal with sales that haven't gone live yet? In that case the end time should be the `sale.liveStartAt`.

Reply:
Created this ticket for this task: https://artsyproduct.atlassian.net/browse/PURCHASE-1584